### DISCUSSION TITLE
Password as a variable

### DISCUSSION THREAD

**Discussion by r-gorajski** [keepassxreboot/keepassxc#9837] [Ideas]
Hello.

I looked through the previous topics, I hope I didn't miss reporting such a functionality.

Would it be possible to add functionality to save the password in KeepassXC as a variable, e.g.: MY_PASS = super secret password?

Let's imagine a situation where I have 10 etries with the same password for different systems (this is a situation where we authenticate ourselves using the Active Directory password, but for some reasons we have to manually enter the AD login and password for several systems). These passwords change, say, every 30 days.

In my opinion, it would be nice if I could create a variable containing the password and provide this variable in the entry configuration.

E.g.: MY_PASS = apw390j[a09w3&^*

In entry:

Title: Robert
Username: robert
Password: {MY_PASS}

**Comment by droidmonkey** [comment]
Please use entry clones and entry references. Here is the documentation: https://keepassxc.org/docs/KeePassXC_UserGuide#_clone_an_entry

**Reply by r-gorajski** [reply]
Excellent! - that's exactly what I had in mind. I'm ashamed that I didn't pay attention to this when I was looking through the documentation.

Thank you very much.
